web-components/src/[sandbox]/sandbox.scss

```diff
@@ -34,6 +34,7 @@
 
 .explorer {  
   width: 100vw;
+  height: calc(100vh - 1.75rem);
 }
```

Comment:
update sandbox so content area doesn't scroll header off screen. makes it easer to switch between themes